cfn/guardduty.template

```diff
@@ -1134,7 +1137,7 @@
                                    ]
                                 },
                                 "\", \"CloudWatchEventsRule\": \"", { "Ref": "CloudWatchEventsRule" },
-                                "\", \"CweRulePattern\": \"{\\\"source\\\":[\\\"aws.guardduty\\\"]}\"",
+                                "\", \"CweRulePattern\": \"{\\\"detail-type\\\":[\\\"GuardDuty Finding\\\"],\\\"source\\\":[\\\"aws.guardduty\\\"]}\"",
```

Comment:
If we make this change then all existing collectors will report error status.

Reply:
or actually not. Should be ok since this is an env variable set via a template.

Reply:
@kkuzmin From testing it seems to be okay.  My stack yesterday was reverted back to the master version of the lambda function so this template even works with the current lambda function, afaics.